pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java

```diff
@@ -179,10 +179,15 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
                     .map(Supplier::get);
 
             SimpleDateFormat dateFormat = new SimpleDateFormat("dd MMMMM yyyy");
+
+            String workspacePath = moduleVersion.getWorkspace().getPath();
+            String attributeFileRelPath =  moduleVersion.getWorkspace().attributeFile().get();
```

Comment:
Small nitpick, but could you remove the double-spacing after the equal sign?

`...elPath =  modu...` <-- double-spacing here.